src/utils/graphql/mutations/updateReview.mutation.ts

```diff
@@ -0,0 +1,16 @@
+import { gql } from '@apollo/client'
+
+export default gql`
+  mutation updateReviewResponses(
+    $id: Int!
+    $reviewResponses: [ReviewResponseOnReviewResponseForReviewResponseReviewIdFkeyUsingReviewResponsePkeyUpdate!]
```

Comment:
🤣

Reply:
I wonder what the longest one we'll get before the end of the project!

Reply:
Hahaha! It's so odd, but that's the way it is. Typescript can be very hard, at least we have been using tools that help a lot with that, I wonder how bad it would have been without it.